witness/src/Dscp/CommonCLI.hs

```diff
@@ -38,11 +38,25 @@ import Dscp.Util
 import Dscp.Web (NetworkAddress (..), ServerParams (..), parseNetAddr)
 import Paths_disciplina_witness (version)
 
+{- |
+[Note default-cli-params]
+
+Several parsers in this file are changed in order to not
+yield default values when no related CLI arguments are provided.
+Parsers yielding default values always override corresponding values
+in config file, making using config files without CLI params at all
+effectively impossible.
+
+To support default values of configuration parameters nevertheless,
+default config values are provided in respective `*.Config` modules.
+-}
+
 logParamsParser :: Log.Name -> Parser LoggingParams
 logParamsParser lpDefaultName = do
     lpDebug <- logDebugParser
     lpConfigPath <- logConfigParser
-    lpDirectory <- logDirParser
+    -- [Note default-cli-params]
+    lpDirectory <- Just <$> logDirParser
```

Comment:
I don't think this makes sense. Now you have to provide both `--log-config` and `--log-dir` in order to make _any_ of these options take place, and there is no way to specify no file for logs.

Reply:
Okay. I was thinking that leaving `logDirParser` optional would be better because `logConfigParser` should parse successfully to make whole `logParamsParser` take place, but it resulting behaviour may be even more confusing.
At least fix help in `logDirParser`.